Jenkinsfile

```diff
@@ -8,19 +8,20 @@ def targets = [
 
 // Map toolchains to compiler labels on Jenkins
 def toolchains = [
-  //ARM: "armcc",
+  ARM: "armcc",
   //IAR: "iar_arm",
   GCC_ARM: "arm-none-eabi-gcc"
   ]
 
 // Create a map of predefined build steps 
-stage "generate build steps for parallel execution"
 def parallelSteps = mbed.createParalleSteps("mbed-os", targets, toolchains)
 
 // Run build steps parallel, map as paramater
-stage "build all targets"
 mbed.compile(parallelSteps)
 
-// Run testapps, mbed-os commit hash or master as parameter
-stage "run mbed-os testapps"
-mbed.runTestApps("${env.GIT_REVISION}")
+def testApps = [
+  "mbed-os-cliapp"
```

Comment:
This cannot be after buildTestApps because it is sent as parameter to the function.